Integrations/Vectra_v2/Vectra_v2_description.md

```diff
@@ -0,0 +1,6 @@
+Vectra API is available for administrators and developers to integrate Vectra's breach detection data
+into their applications. Vectra provides access to security event data, platform
+configuration, and health information via URI paths.
+
+Tested with API version 2.1. 
+c_score and t_score fields changed to Certainty and Threat accordingly (range: 0-100)
```

Comment:
parameters c_score and t_score should be changed as the arguments.

Reply:
this is the update to the previous version users so they will be aware of the change